processor/src/memory/mod.rs

```diff
@@ -209,17 +209,31 @@ impl Memory {
         }
     }
 
-    // TEST METHODS
-    // --------------------------------------------------------------------------------------------
-
-    /// Instantiates a new processor for testing purposes.
-    #[cfg(test)]
+    /// Reads a specific address of it's value.
```

Comment:
"Reads a specific address of it's value." is confusing. Maybe say something like: "Returns a word located at the specified address, or None if the address hasn't been accessed previously."